jmix-gradle-plugin/src/main/groovy/io/jmix/gradle/JmixPlugin.groovy

```diff
@@ -152,4 +155,71 @@ class JmixPlugin implements Plugin<Project> {
         }
         return result ?: 'unspecified'
     }
+
+    private static void registerCleanConfTask(Project project) {
+        project.tasks.register('cleanConf', Delete) {
+            doFirst {
+                def resources = project.file("src/main/resources/")
+                if (resources.exists() && resources.isDirectory()) {
+                    def mainProperties = new Properties()
+                    project.file("src/main/resources/application.properties").withInputStream { mainProperties.load(it) }
+
+                    def confDir = resolveConfDir(project, mainProperties)
+
+                    println("Delete directory = ${confDir}")
+                    delete "${confDir}"
+                } else {
+                    return
+                }
+            }
+        }
+        project.pluginManager.withPlugin("org.springframework.boot") {
+            project.tasks.named("bootRun") {
+                dependsOn("cleanConf")
+            }
+        }
+    }
+
+    private static String resolveConfDir(Project project, Properties mainProperties) {
+        def profilesList = resolveActiveProfiles(project, mainProperties)
+
+        def confDir = null
+        if (!profilesList.isEmpty()) {
+            for (def profileName : profilesList) {
+                println("Check profile: " + profileName)
+                def profilePropertyFilePath = "src/main/resources/application-%s.properties".formatted(profileName)
+                def profilePropertiesFile = project.file(profilePropertyFilePath)
+                if (profilePropertiesFile.exists()) {
+                    def profileProps = new Properties()
+                    project.file(profilePropertyFilePath).withInputStream { profileProps.load(it) }
+                    confDir = profileProps.getProperty("jmix.core.work-dir") ?: profileProps.getProperty("jmix.core.workDir") ?: null
```

Comment:
Why "work-dir"? We have a separate property for conf dir: https://docs.jmix.io/jmix/app-properties.html#jmix.core.conf-dir

Reply:
Mixed up at some point. Fixed.